sdk/python/kfp/dsl/_container_op.py

```diff
@@ -155,6 +164,16 @@ def set_cpu_limit(self, cpu):
     self._validate_cpu_string(cpu)
     self.cpu_limit = cpu
 
+  def set_nvidia_gpu_limit(self, gpu):
```

Comment:
Not sure how many types of gpus we will support but I am under the impression that nvidia is the mainstream one. Would it be better to do:

set_gpu_limit(self, num_gpu, vendor='nvidia')?

Reply:
I hope we don't make assumption here. e.g. user might want to use amd one.

Reply:
Okay.